### PR TITLE
Extend partest guide with sbt and note removals

### PR DESCRIPTION
**Pull request by janekdb**
Based on comments by @hrhino and @martijnhoekstra.

### CODE REVIEW

File: contribute/partest-guide.md [hunk 1]
```diff
@@ -3,15 +3,35 @@ layout: page
...
+
+You can get a summary of the usage by running `partest --help`.
+
+If you would like to run particular tests pass the test paths as arguments
+
+        > partest test/files/pos/bounds.scala test/scaladoc/run/diagrams-base.scala
+
+To run only the Scaladoc tests use `--srcpath` with the location of the tests
+
+        > partest --srcpath scaladoc
```
Comment (hrhino) [inline contribute/partest-guide.md]: You could also mention that `partest pos` works, and executes all of the `pos` tests (likewise for `neg`, `run`, `jvm`, &c.).

Comment (janekdb) [inline contribute/partest-guide.md]: Definitely has a benefit but I'm happy to leave a fuller update to a separate PR. This is about plugging a gap related to 2.12 and sbt rather than documenting partest.
